ghost/core/test/unit/server/models/stripe-customer-subscription.test.js

```diff
@@ -25,33 +25,6 @@ describe('Unit: models/stripe-customer-subscription', function () {
             sinon.restore();
         });
 
-        it('ignores trial details when flag is off', function () {
-            const trialStartAt = new Date();
-            const trialEndAt = new Date(Date.now() + 7 * 86400 * 1000); // trial ending 7 days from now
-            const stripeSubscription = {
-                customer_id: 'fake_customer_id',
-                subscription_id: 'fake_subscription_id',
-                plan_id: 'fake_plan_id',
-                stripe_price_id: 'fake_plan_id',
-                plan_amount: 1337,
-                plan_nickname: 'e-LEET',
-                plan_interval: 'year',
-                plan_currency: 'btc',
-                status: 'active',
-                start_date: new Date(),
-                current_period_end: new Date(),
-                cancel_at_period_end: false,
-                trial_start_at: trialStartAt,
-                trial_end_at: trialEndAt
-            };
-
-            const json = serialize(stripeSubscription);
-            should.exist(json.plan);
-            should.exist(json.customer);
-            should.not.exist(json.trial_start_at);
-            should.not.exist(json.trial_end_at);
-        });
-
         it('returns trial details with flag enabled', function () {
```

Comment:
nitpick but can we fix the name of this test and remove the following line now? Saves cleanup later, especially given that it doesn't say which flag this refers to - we'll likely lose this!

Reply:
Ah yeah sure, good point! On it.

Reply:
Done ✅